pkg/resourcegenerator/istio/service_entry.go

```diff
@@ -14,9 +14,43 @@ import (
 	"strings"
 )
 
+func setCloudSqlRule(accessPolicy *podtypes.AccessPolicy, object client.Object) (*podtypes.AccessPolicy, error) {
+	application, ok := object.(*skiperatorv1alpha1.Application)
+	if !ok {
+		return accessPolicy, nil
+	}
+
+	if !util.IsCloudSqlProxyEnabled(application.Spec.GCP) {
+		return accessPolicy, nil
+	}
+
+	if application.Spec.GCP.CloudSQLProxy.IP == "" {
+		return nil, errors.New("cloud sql proxy IP is not set")
+	}
+
+	externalRule := &podtypes.ExternalRule{
+		Host:  fmt.Sprintf("%x.cloudsql", util.GenerateHashFromName(application.GetName())),
```

Comment:
I would love to have something more descriptive, just for making debugging Istio easier. I understand that this field is set in order to make Istio validation pass, but this hostname might appear in the logs. What about `appname-${hash(appName-image:tag}` in order to make them more distinct across various deployed versions of the app (e.g. preview environments)?

Reply:
I changed the name to `appName-image.cloudsql` so its more descriptive.

Reply:
done :+1: